states/apache2/tls.sls

```diff
@@ -0,0 +1,20 @@
+include:
+  - apache2
+  - apache2.mod_ssl
+  - tls
+
+
+{{ sls }} installed packages:
+  pkg.installed:
+    - pkgs:
+      - apache2
+    - require:
+      - file: tls dhparams.pem
+{%- if pillar.mounts %}
+{%- for mount in pillar.mounts %}
+      - mount: mount mount {{ mount.file }}
+{%- endfor %}
+{%- endif %}
```

Comment:
Is it possible to avoid having these lines in both this file, and the included init (for apache2)?  If so, that might avoid a potential for the files going out of sync in the future.